### ISSUE TITLE
UV380G can tx but not rx on 477.125MHz

### ISSUE THREAD

**Issue by vk7js** [OpenRTX/OpenRTX#112]
When testing OpenRTX on my UV380 with GPS, It seems that frequencies around 476-477 MHz, the Australian CB frequency range,  (within the 400-480 MHZ range the radio is supposed to support) the radio will tx but not rx on fm mode with and without CTCSS tones set.

**Comment by silseva** [comment]
I'm going to investigate this soon by sniffing what the OEM firmware sends to the radio chip

**Comment by blazeb5** [comment]
Hello,  I'm in the 438.800.000mhz on my hotspot and radios. I've moded an rt3s and uv380. The rt3s works fine but my uv 380 will not rx from the other radio or hotspot. I'm running the nightly build from 23oct2023 and same results with v.3.6 stable build of OPEN RTX...any help/advice?

**Comment by chrisdebian** [comment]
@vk7js Do you know if this is still an issue?

**Comment by silseva** [comment]
Yes, is still an issue hence its presence in the issues list. The code for the radio driver has been written by reverse engineering the OEM firmware, which makes it easy for these kind of bugs to slip in. Because of this, finding where exactly the issue is is extremely time consuming: for the time being we decided to concentrate our (scarce) resources in implementing missing but fundamental features.
This issue will be investigated later, during the firmware consolidation phase.
